contracts/makefile

```diff
@@ -113,6 +113,25 @@ interact-sepolia :;
 	@echo "Interacting through $(contract)..."
 	@forge script scripts/interactions/${contract}.s.sol:${contract} --ffi --rpc-url sepolia --private-key ${TESTNET_PRIVATE_KEY} --broadcast --legacy
 
+# ================================== ALPHA ==================================
+
+deploy-alpha-base-anvil:;
+	@echo "Deploying $(contract)..."
+	@SAVE_DEPLOYMENTS=1 DEPLOYMENT_CONTEXT=alpha forge script scripts/deployments/$(if $(findstring facet,$(type)),facets/)${contract}.s.sol:${contract} --ffi --rpc-url base_anvil --private-key ${LOCAL_PRIVATE_KEY} --broadcast
+
+deploy-alpha-sepolia :;
+	@echo "Deploying $(contract)..."
+	@SAVE_DEPLOYMENTS=1 DEPLOYMENT_CONTEXT=alpha forge script scripts/deployments/$(if $(findstring facet,$(type)),facets/)${contract}.s.sol:${contract} --ffi --rpc-url sepolia --private-key ${TESTNET_PRIVATE_KEY} --broadcast --verify --legacy -vvvv
+
+deploy-alpha-base-sepolia :;
+	@echo "Deploying $(contract)..."
+	@SAVE_DEPLOYMENTS=1 DEPLOYMENT_CONTEXT=alpha forge script scripts/deployments/$(if $(findstring facet,$(type)),facets/)${contract}.s.sol:${contract} --ffi --rpc-url base_sepolia --private-key ${TESTNET_PRIVATE_KEY} --broadcast --verify --verifier blockscout --verifier-url ${BLOCKSCOUT_SEPOLIA_URL}
+
+deploy-alpha-river-testnet :;
```

Comment:
Are you planning on including alpha riverRegistry deployment addresses under packages/generated/deployments/alpha/river in this PR ? This is needed for packages preinstall hook to succeed on a build.

Reply:
should be up now.